### PR TITLE
Fix player associations

### PR DESCRIPTION
**Pull request by kirubel**
Fixed black and white player associations
Set white player as current user when creating a new game

### CODE REVIEW

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -9,9 +9,11 @@ def new
   end
 
   def create
-  	@game = Game.create(game_params)
+  	@game = Game.new(game_params)
+    @game.white_player = current_user
+    @game.save!
   	redirect_to game_path(@game)
-    # how do we set white_player_id to current user_id?
+
```
Comment (washburnad) [inline app/controllers/games_controller.rb:16]: What happens if game.save fails?  Also, I'd love to see a corresponding test to back up the code.

**Comment by ghost** [comment]
Nice. Kirubel. I see what you were trying to do. I had delete the white_player & black_player associations because I didn't understand that was the only way to set up the appropriate user id. 

Only question I have is similar to Anna's. Don't we need to keep has_many :user in the game.rb model?

Other than that- I'm good to merge. We should make a note about Aaron's game.save test though.

**Comment by washburnad** [comment]
@David, don't merge until there's a test! Tested code should be a prerequisite to merging.
As far as the games - users relationship, engage the group in a discussion.  Run your rails console locally and try accessing games for a particular user, users for a particular game, etc.. Play with it to see how the different relationships differ.

**Comment by ghost** [comment]
Oh good point Aaron. Ok I'm going to try an figure out how run rails
console locally and do this.

Do I type these type of things into my terminal?

rails console

game = Game.new

etc?

**Pull request merged**